doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -222,6 +222,97 @@ gls{Palladio}
  runs on.
 \end_layout
 
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ20/"
+
+\end_inset
+
+ The measurements run without user interaction.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ30/"
+
+\end_inset
+
+ The measurements also shall run over the network e.g.
+ on a Server.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ40/"
+
+\end_inset
+
+ The user has the possibility to pause the measurements.
+ As well the user can continue a measurement from save.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ50/"
+
+\end_inset
+
+ If it is requested, Beagle shuts down the PC after it finished.
```

Comment:
This is a optional **functional** requirement. But since I'm usually wrong when it comes to the question whether something is functional or non-functional: @someone else: Please state your opinion. In mine, it's functional since this would add another function.